src/components/Modal.mock.ts

```diff
@@ -0,0 +1,29 @@
+import { FunctionComponent } from 'react'
+
+
+export type ModalExports = typeof import('src/components/Modal') & { __esModule: true }
+
+/**
+ * provides a mocked version of Modal module's Modal component that avoids
+ * pitfalls specific to missing services in jest simulated browser environment
+ * including dom measurement calls.
+ */
+export const mockModalModule = (): ModalExports => {
+  const originalModule = jest.requireActual<ModalExports>('src/components/Modal')
+
+  type ModalFn = FunctionComponent & {propTypes: typeof originalModule.modalPropTypes};
+
+  // Stub out onAfterOpen for noOp because real implementation needs
+  // unavailable dom measuring services
```

Comment:
I don't think this comment is accurate. The default `onAfterOpen` function deals with moving focus.

Reply:
hmm.... curious

Reply:
I can add some of this detail to the mock file...